src/css/profile/mobile/changeable/common/progress.less

```diff
@@ -97,9 +97,10 @@ tau-progress {
 .ui-progress-circle {
 	position: relative;
 	display: inline-block;
-	pointer-events: none;
 	box-sizing: border-box;
-
+	* {
```

Comment:
This change make sense.
`Elements with pointer-events set to none will be ignored, and the element below it will be returned.`
But I have to check possible negative effects.